tests/suites/backup_restore/backup_restore.sh

```diff
@@ -0,0 +1,71 @@
+run_basic_backup_create() {
+    set -e  # TODO benhoyt: remove once "set +e" issue is fixed
+    echo
+
+    juju switch controller
```

Comment:
I'm assuming that you can only backup from a controller? If so can we add a note about why you're not using ensure here..?

Reply:
Yes - clarified with comment. Not sure why I would use "ensure" here?

Reply:
Ah, I see what you mean by "ensure" now -- fixed, thanks.